backend/usecases/chat/tool_executor.py

```
@@ -130,6 +131,8 @@ async def execute_loop(
                 )
             except asyncio.TimeoutError:
                 logger.error("LLM request timed out")
```

Ruff (0.14.13)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)